games-arcade/opensyobonaction/opensyobonaction-2.rc2.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="An open-source cross-platform port of the Japanese platformer using SDL"
```

Comment:
If I'm not mistaken, the line with the SUMMARY variable should be <= 70 chars.
I don't remember, however, if that's the limit for the whole line or just for the content of the variable.
I should have looked for the info, but I was affraid of comming back too late...
I think the limit is for the whole line, not counting just the newline. If someone could confirm that would be great ;-)